requirements/testing_sheet.txt

```diff
@@ -0,0 +1,6 @@
+
+# These are the requirements to create an Integration testing Spreadsheet
+
+oauth2client
+PyOpenSSL
```

Comment:
these should be pinned to specific versions

Reply:
I'll do it thanks.